common/source/docs/common-team.rst

```diff
@@ -8,29 +8,43 @@ Team
 The ArduPilot Dev Team
 ======================
 
-Please check out our lists for **ArduPilot**
-`contributors <https://github.com/ArduPilot/ardupilot/graphs/contributors>`__
-and `maintainers <https://github.com/ArduPilot/ardupilot#maintainers>`__
-on GitHub.
+The **ArduPilot Project**  is `maintained by <https://github.com/ArduPilot/ardupilot#maintainers>`__ 
+individuals from all over the world, who are responsible for managing and reviewing submissions to
+their respective parts of what is a very significant endeavour.
+
+As a global project with many elements, ArduPilot is supported by many contributors, some of whom are listed below:
+
+**ArduPilot** vehicle code `contributors <https://github.com/ArduPilot/ardupilot/graphs/contributors>`__
 
 **Mission Planner GCS** `contributors <https://github.com/ArduPilot/MissionPlanner/graphs/contributors>`__
 
 **APM Planner 2.0 GCS** `contributors <https://github.com/ArduPilot/apm_planner/graphs/contributors>`__ 
 
+**MAVProxy GCS** `contributors <https://github.com/ArduPilot/MAVProxy/graphs/contributors>`__ 
+
+**Companion Computing** `contributors <https://github.com/ArduPilot/companion/graphs/contributors>`__
+
+**ArduPilot Documentation** `contributors <https://github.com/ArduPilot/ardupilot_wiki/graphs/contributors>`__ 
+
 .. note::
 
-   Hundreds of people have contributed to ArduPilot project over
+   Hundreds of people have contributed to the ArduPilot project over
    time, in ways large and small. Sadly not all of them will be listed
    above.
 
 Collaboration with other projects
 =================================
 
-The ArduPilot dev team is also fortunate to benefit from collaboration
+The ArduPilot team is fortunate to have benefited from collaboration
 with some related projects, including:
 
--  The `PX4 dev team <http://pixhawk.org/en/start>`__
--  The `MatrixPilot team <https://github.com/MatrixPilot/MatrixPilot/wiki>`__
+-  The `PX4 dev team <https://github.com/PX4/Firmware/graphs/contributors>`__
+-  The `QGroundControl team <https://github.com/mavlink/qgroundcontrol/graphs/contributors>`__
+-  The `MAVLink team <https://github.com/mavlink/mavlink/graphs/contributors>`__
+-  The `DroneKit team <https://github.com/dronekit/dronekit-python/graphs/contributors>`__  
+-  The `Paparazzi dev team <https://github.com/paparazzi/paparazzi/graphs/contributors>`__
+-  The `MatrixPilot team <https://github.com/MatrixPilot/MatrixPilot/graphs/contributors>`__
```

Comment:
I think we should link the projects and not the contributors graphs. Just because someone contributed to these other projects doesn't mean they have contributed anything to ArduPilot.